Essentials/src/com/earth2me/essentials/commands/Commandeco.java

```diff
@@ -53,6 +54,9 @@ public void run(final Server server, final CommandSource sender, final String co
 
     @Override
     protected void updatePlayer(final Server server, final CommandSource sender, final User player, final String[] args) throws NotEnoughArgumentsException, ChargeException, MaxMoneyException {
+        if(isPercent && cmd != EcoCommands.RESET) {
```

Comment:
Forgot, there should also be a space between the `if` and `(` here. By the way, using CTRL+ALT+L in IntelliJ will auto format the file for you, in standard formatting (which is what is used here).